### PR TITLE
Add Route Usage measuring

### PR DESCRIPTION
**Pull request by TomasVotruba**
Inspired by https://github.com/julienbourdeau/route-usage/

After propper testing, decopule and move to @migrify project

<br>

Thanks @julienbourdeau for clear code to learn from and great idea! Any feedback appreciated :)

### CODE REVIEW

File: .github/workflows/integration_tests.yaml [hunk 1]
```diff
@@ -7,6 +7,17 @@ jobs:
...
         runs-on: ubuntu-latest
 
+        services:
+            postgres:
+                image: postgres:11.7
+                env:
+                    POSTGRES_PASSWORD: root
+                    POSTGRES_DB: pehapkari_cz
+                ports:
+                    - 5432:5432
```
Comment (TomasVotruba) [inline .github/workflows/integration_tests.yaml:17]: Interesting... it looks exactly like Docker configuration

File: .github/workflows/integration_tests.yaml [hunk 2]
```diff
@@ -17,7 +17,6 @@ jobs:
 
             -   run: composer install --no-progress
             -   run: |
-                    # we need to run this in Docker to get into the database that started in Docker
-                    docker run --rm pehapkari/pehapkari.cz:${TRAVIS_COMMIT} php bin/console doctrine:database:create --env=test
-                    docker run --rm pehapkari/pehapkari.cz:${TRAVIS_COMMIT} php bin/console doctrine:schema:create --env=test
-                    docker run --rm pehapkari/pehapkari.cz:${TRAVIS_COMMIT} php vendor/bin/phpunit --testsuite integration
+                    bin/console doctrine:database:create --env=test
+                    bin/console doctrine:schema:create --env=test
+                    php vendor/bin/phpunit --testsuite integration
```
Comment (TomasVotruba) [inline .github/workflows/integration_tests.yaml]: Much preffered :+1:

File: .github/workflows/integration_tests.yaml [hunk 3]
```diff
@@ -28,6 +30,6 @@ jobs:
 
             -   run: composer install --no-progress
             -   run: |
-                    bin/console doctrine:database:create --env=test
-                    bin/console doctrine:schema:create --env=test
+                    # We should run migrations instead, because we can check afterwards if migrations are up to date
+                    bin/console doctrine:schema:create
```
Comment (TomasVotruba) [inline .github/workflows/integration_tests.yaml]: How is the database created now?

Comment (JanMikes) [reply .github/workflows/integration_tests.yaml]: https://github.com/pehapkari/pehapkari.cz/blob/master/.docker/docker-entrypoint.sh#L26

It is not handled very well, we should change it to migrations in future :-)

File: .github/workflows/integration_tests.yaml (hunk 2 above)
Comment (JanMikes) [reply .github/workflows/integration_tests.yaml]: Glad to help 😉 i know this is much easier to read

File: .github/workflows/integration_tests.yaml (hunk 1 above)
Comment (JanMikes) [reply .github/workflows/integration_tests.yaml:17]: It is very close to it. Source: https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idservices 

You know, yamls everywhere 😸

**Pull request merged**